app/models/illustrator_printing.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class IllustratorPrinting < ApplicationRecord
+  self.table_name = "illustrators_printings"
+
+  belongs_to :illustrator,
+    :primary_key => :id,
```

Comment:
Just in case you don't know, you can write hashs that have symbol keys as `primary_key: :id`, without using the hash rocket. See [here](https://stackoverflow.com/a/8675229/3023252), for example.

Reply:
ah, nice.  thanks.